app/API/V1/Repositories/EmailVerificationRepository.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace App\API\V1\Repositories;
+
+use App\API\V1\Entities\User;
+use App\API\V1\Entities\Permission;
+use App\API\V1\Entities\EmailVerification;
+use App\Repositories\Repository;
+
+/** @noinspection LongInheritanceChainInspection */
+class EmailVerificationRepository extends Repository
+{
+    protected /** @noinspection ClassOverridesFieldOfSuperClassInspection */
+        $entity = EmailVerification::class;
+
+    /**
+     * @param User $user
+     * @return EmailVerification
+     * @throws \Doctrine\DBAL\ConnectionException
+     */
+    public function createEmailVerificationCode(User $user): EmailVerification
```

Comment:
When we screen share I will show you how we can do this in a very scribe kind of way with very few lines of code so it can be an example to people using the skeleton.